modules/gov/msgs_test.go

```diff
@@ -39,8 +39,10 @@ func TestMsgSubmitProposal(t *testing.T) {
 		{"Test Proposal", "the purpose of this proposal is to test", ProposalTypeText, addrs[0], coinsMulti, true},
 	}
 
+	var params Params
```

Comment:
We should create a new folder "params" under the "gov", and put all these source files in it

Reply:
Ok, I will do it tomorrow.